tests/integration/it_director_privilages_test.py

```diff
@@ -71,3 +86,89 @@ def test_cant_access_other_company_employees(app, db_session):
         method=Method.READ, resource="employee", employee_id=other.id
     )
 
+
+def test_can_access_subalterns(app, db_session):
```

Comment:
@amihaiemil I suggest to use factories to make the code simpler and shorter, isee how it's done in https://github.com/timelesslounge/timelessis/blob/master/tests/integration/it_manager_privilages_test.py#L94